.github/workflows/check-image.yml

```diff
@@ -4,12 +4,21 @@ on:
     - cron: "5 0 * * *"
   workflow_dispatch:
 
+permissions:
```

Comment:
@samdoran can correct me if I am wrong, but now that konflux is configured, I think we can delete this workflow entirely.

Reply:
My understanding is the purpose of this workflow is to prompt an image rebuild when the packages in the base container have changed. Moving to Konflux doesn't really change the need to rebuild the image when the base image changes. There's nothing special in Konflux to rebuild when the base image changes.

This project is high enough tempo that I think this workflow is unnecessary and can be removed.

For other projects that are lower tempo, this workflow is more useful. For those lower tempo projects, we could rely on the Konflux pipeline update PRs from Renovate bot as an indirect means to accomplish the same thing.

Reply:
ok!